src/modlist.cpp

```diff
@@ -223,8 +197,7 @@ QVariant ModList::data(const QModelIndex &modelIndex, int role) const
       return version;
     }
     else if (column == COL_PRIORITY) {
-      int priority = modInfo->getFixedPriority();
-      if (priority != INT_MIN) {
+      if (modInfo->isBackup() || modInfo->isOverwrite()) {
```

Comment:
Why not `isFixedPriority()`?

Reply:
Because I added `isFixedPriority` after and forgot this part, will fix.